roles/pcp/tasks/web.yml

```diff
@@ -1,8 +1,8 @@
 ---
 - name: Fail when on Ubuntu
   fail:
-    msg: "pcp-web isn't available on Ubuntu yet :("
-  when: ansible_distribution == "Ubuntu"
+    msg: "pcp-webapi is only available when using upstream packages. Set upstream_repo to true."
+  when: ansible_distribution == "Ubuntu" and upstream_repo|bool != true
```

Comment:
Might be time to start double-quoting variables in new/modified code since Ansible v2 throws a warning for each unquoted variable.